src/restriction_level.rs

```diff
@@ -58,13 +58,21 @@ impl RestrictionLevelDetection for &'_ str {
             }
         }
 
+        fn single_item<I: Iterator>(mut iter: I) -> Option<I::Item> {
```

Comment:
Can we instead implement this via `count_ones()`?

Reply:
Do you mean accessing the bitset underlying `ScriptExtension` directly? We cannot access the implementation detail from this crate...

Reply:
Right, add that to the original crate. Maybe a `len()` method

Reply:
The PR is merged, now we need a new release of that crate.

Reply:
Done.